lighthouse-core/runner.js

```diff
@@ -202,6 +210,7 @@ class Runner {
     // Fill remaining audit result fields.
     }).then(auditResult => Audit.generateAuditResult(audit, auditResult))
     .catch(err => {
+      log.warn(audit.meta.name, `Caught exception: ${err.message}`);
```

Comment:
this is going to log gatherer errors twice, though. Not sure if we should care though (since gatherer errors are audit errors too, sort of).